ts_src/script.ts

```diff
@@ -173,7 +173,8 @@ export function fromASM(asm: string): Buffer {
   return compile(
     asm.split(' ').map(chunkStr => {
       // opcode?
-      if (OPS[chunkStr] !== undefined) return OPS[chunkStr];
+      if (OPS[chunkStr as keyof typeof OPS] !== undefined)
```

Comment:
I think this kind of proves the "compatible way" part not being 100% accurate.

If we look at the JS, then yeah. But TypeScript people will have breakage.

Reply:
This is a great PR though.

I'm thinking of how to push this through.

Reply:
Not sure how many people are writing script parsers from scratch just pulling in the OPS though...

The use case of "get opcode from arbitrary string" is probably very niche. But still it's a possible breakage.